cg/services/analysis_starter/configurator/file_creators/utils.py

```diff
@@ -0,0 +1,49 @@
+import logging
+from pathlib import Path
+
+from cg.constants import FileExtensions
+from cg.constants.nf_analysis import NextflowFileType
+from cg.io.csv import write_csv
+from cg.io.json import write_json
+from cg.io.yaml import write_yaml_nextflow_style
+from cg.services.analysis_starter.configurator.file_creators.abstract import FileContentCreator
+
+LOG = logging.getLogger(__name__)
+
+# TODO: Adapt to gene panel and variant files
+FILE_TYPE_TO_EXTENSION: dict[NextflowFileType, FileExtensions] = {
```

Comment:
```suggestion
FILE_TYPE_TO_EXTENSION: dict[NextflowFileType, str] = {
```
I think this is the proper typehint since you assign a str and not the FileExtension class